apps/nextjs-website/src/lib/webinars/webinarQuestions.ts

```diff
@@ -30,6 +31,10 @@ export type WebinarQuestion = {
   readonly webinarId: string;
   readonly question: string;
   readonly createdAt: Date;
+  readonly hiddenBy?: string;
+  readonly hiddenByFullName?: string;
+  readonly highlightedBy?: string;
+  readonly highlightedByFullName?: string;
```

Comment:
```suggestion
  readonly hiddenBy?: string;
  readonly highlightedBy?: string;
```
I would keep only these parameters and I would fill them with the full name.
According to the activity on Jira, the requirements is to persist the full name.